docs/website/tds/tutorial/CatalogPrimer.html

```diff
@@ -436,7 +436,7 @@ <h2><a name="Using_ToolsUI">Using ToolsUI to view catalogs</a></h2>
 
     <p>The NetCDF Tools User Interface (aka ToolsUI) can read and display THREDDS catalogs. You can start it from the
         command line, or
-        <a href="https://www.unidata.ucar.edu/software/netcdf-java/v4.6/webstart/netCDFtools.jnlp">launch it
+        <a href="https://www.unidata.ucar.edu/netcdf-java/4.6/userguide/webstart/netCDFtools.jnlp">launch it
```

Comment:
Let's just reference the URL on the docs server directly and not rely on the redirect we will put into place.

Reply:
webstart links updated to point to downloads.unidata.ucar.edu.